schema/synthetix/view_synths_rates.sql

```diff
@@ -0,0 +1,30 @@
+BEGIN;
```

Comment:
added two materialized views here to speed up performance. I can't test it though. Let me know if it needs any changes.